src/index.html

```diff
@@ -5,8 +5,393 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__logo-link"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+        >
+      </a>
+
+      <nav class="header-menu">
+        <ul class="header-menu__links-list">
+          <li class="header-menu__list-item">
+            <a
+              class="header-menu__menu-item is-active"
+              href="#"
+            >
+              Apple
+            </a>
+          </li>
```

Comment:
```suggestion
      <nav class="navigation">
        <ul class="navigation__list">
          <li class="navigation__item">
            <a
              class="navigation__link is-active"
              href="#"
            >
              Apple
            </a>
          </li>
```

ul in nav is always fulfilled with links, so it's not really neccesary to add prefix 'links'. each li in ul is element of your block, so you call them blockName__item, in our example it's navigation__item, and each link in our navigation is blockName__link, so we name them as navigation__link. I just wrote my example of naming, how to name them is up to you, but you must me attentive when you name elements, because in your example of code it's not appropriate naming style, you should check video about BEM again, just to make sure you understood theme right. Tried to explain as good as I understand it.